test/e2e/cypress/support/index.ts

```diff
@@ -6,3 +6,7 @@
 // ***********************************************************
 
 import 'mattermost-webapp/e2e/cypress/tests/support';
+
+before(() => {
+    localStorage.setItem('__landingPageSeen__', String(true));
```

Comment:
Here's the landing page fix. The webapp checks for this value and avoids showing the landing page screen if it's set.